diagnostics/forcing_feedback/forcing_feedback_kernelcalcs.py

```diff
@@ -0,0 +1,337 @@
+#    and the MDTF code package. See LICENSE.txt for the license. 
+#
+
+import os
+import sys
+import numpy as np
+import xarray as xr
+import pandas as pd
+import netCDF4
+from scipy.interpolate import interp1d
+from scipy.interpolate import griddata
+
+#Import functions specific to this toolkit
+from forcing_feedback_util import fluxanom_calc_4D
+from forcing_feedback_util import fluxanom_calc_3D
+from forcing_feedback_util import esat_coef
+from forcing_feedback_util import latlonr3_3D4D
+from forcing_feedback_util import fluxanom_nc_create
+from forcing_feedback_util import feedback_regress
+# ======================================================================
+
+#Read in radiative kernels
+kernnc = xr.open_dataset(os.environ["OBS_DATA"]+"/forcing_feedback_kernels.nc")
+lat_kern = kernnc['latitude'].values
+lon_kern = kernnc['longitude'].values
+lev_kern = kernnc['plev'].values
+time_kern_TOA = kernnc['time'].values
+lw_t_kern_TOA = kernnc['lw_t'].values #LW all-sky air temperature kernel
+lwclr_t_kern_TOA = kernnc['lwclr_t'].values #LW clear-sky air temperature kernel
+lw_ts_kern_TOA = kernnc['lw_ts'].values #LW all-sky surface temperature kernel
+lwclr_ts_kern_TOA = kernnc['lwclr_ts'].values #LW clear-sky surface temperature kernel
+lw_q_kern_TOA = kernnc['lw_q'].values #LW all-sky water vapor radiative kernel
+lwclr_q_kern_TOA = kernnc['lwclr_q'].values #LW clear-sky water vapor radiative kernel
+sw_q_kern_TOA = kernnc['sw_q'].values #SW all-sky water vapor radiative kernel
+swclr_q_kern_TOA = kernnc['swclr_q'].values #SW clear-sky water vapor radiative kernel
+sw_a_kern_TOA = kernnc['sw_a'].values #SW all-sky surface albedo radiative kernel
+swclr_a_kern_TOA = kernnc['swclr_a'].values #SW clear-sky surface albedo radiative kernel
+ps_kern_TOA = kernnc['PS'].values #Radiative kernel surface pressure
+
+#close kernel file
+kernnc.close()
+
+
+#Read in model output
+os.environ["ts_file"] = os.environ["CASENAME"]+"."+os.environ["ts_var"]+".mon.nc"
+os.environ["ta_file"] = os.environ["CASENAME"]+"."+os.environ["ta_var"]+".mon.nc"
+os.environ["hus_file"] = os.environ["CASENAME"]+"."+os.environ["hus_var"]+".mon.nc"
+os.environ["rsus_file"] = os.environ["CASENAME"]+"."+os.environ["rsus_var"]+".mon.nc"
+os.environ["rsds_file"] = os.environ["CASENAME"]+"."+os.environ["rsds_var"]+".mon.nc"
+os.environ["rsuscs_file"] = os.environ["CASENAME"]+"."+os.environ["rsuscs_var"]+".mon.nc"
+os.environ["rsdscs_file"] = os.environ["CASENAME"]+"."+os.environ["rsdscs_var"]+".mon.nc"
+os.environ["rsdt_file"] = os.environ["CASENAME"]+"."+os.environ["rsdt_var"]+".mon.nc"
+os.environ["rsut_file"] = os.environ["CASENAME"]+"."+os.environ["rsut_var"]+".mon.nc"
+os.environ["rsutcs_file"] = os.environ["CASENAME"]+"."+os.environ["rsutcs_var"]+".mon.nc"
+os.environ["rlut_file"] = os.environ["CASENAME"]+"."+os.environ["rlut_var"]+".mon.nc"
+os.environ["rlutcs_file"] = os.environ["CASENAME"]+"."+os.environ["rlutcs_var"]+".mon.nc"
```

Comment:
These definitions should be removed: absolute paths to all requested variables are set via the `path_variable` setting in the new settings file format, which defaults to `<variable name>_FILE`; see [documentation](https://mdtf-diagnostics.readthedocs.io/en/latest/sphinx/ref_settings.html#varlist-entry-properties).